store/datastore/utils.go

```diff
@@ -38,34 +39,33 @@ func rebind(query string) string {
 
 // helper function that converts a simple repsitory list
 // to a sql IN statment.
-func toList(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "?"
-		in[i] = repo.FullName
-	}
-	return strings.Join(qs, ","), in
-}
+func toList(listof []*model.RepoLite) ([]string, [][]interface{}) {
+	var total = len(listof)
```

Comment:
This was my first solution... It did not seem attractive for me because the limit variable and the core logic for pagination was replicated on the repos and users functions.